packages/web-components/src/components/ic-navigation-group/ic-navigation-group.tsx

```diff
@@ -481,53 +501,47 @@ export class NavigationGroup {
           "ic-navigation-group-collapsed": !this.expanded,
           ["ic-navigation-group-side-nav"]: this.navigationType === "side",
           [`ic-theme-${this.theme}`]: this.theme !== "inherit",
+          ["expandable"]: !!expandable,
```

Comment:
minor on, but should this be made more unique (eg include the component name : `ic-navigation-group-expandable`). we previously changed class names applied on the host, so there is less risk of clashing with names users may apply

Reply:
Sorted 👍